packages/firebase_auth/firebase_auth_web/lib/firebase_auth_web.dart

```diff
@@ -280,9 +285,10 @@ class FirebaseAuthWeb extends FirebaseAuthPlatform {
   @override
   Future<PlatformAuthResult> signInWithEmailAndLink(
       String app, String email, String link) async {
-    // TODO(hterkelsen): Use signInWithEmailLink once 7.0.0 of package:firebase
-    // is released.
-    throw UnimplementedError('signInWithEmailAndLink');
+    final firebase.Auth auth = _getAuth(app);
+    final firebase.UserCredential userCredential =
+        await auth.signInWithEmailLink(email, link);
```

Comment:
This might _"Fail with an error if the email address is invalid or OTP in email link expires."_. [Docs](https://firebase.google.com/docs/reference/js/firebase.auth.Auth.html#signinwithemaillink).

Does that case need to be handled, or would that exception (?) propagate to the client app? _(I've seen other places in the web plugin with a similar pattern... How do the Android/iOS versions behave in that case?)_

Reply:
Right now it looks like the function will throw a `FirebaseError` instance to the client which we will want to map to a `PlatformException` but that would require #1698 to land first.

Reply:
Ah, yes, that 1698 is going to be handy. Thanks for pointing it out!